src/component/framework/actions.js

```diff
@@ -1,7 +1,7 @@
 export class Actions {
 
-  activate(model) {
-    this.value = model.value;
+  activate(vm) {
+    this.vm = vm;
```

Comment:
By doing this i mutate the form-field's model, thus I'm able to mutate the model or value there.